css/components/main.css

```diff
@@ -0,0 +1,6 @@
+.main {
```

Comment:
This is the `tile-wrapper` style. I think leaving out the `div` and styling the main directly is a better solution.